### ISSUE TITLE
Source CertSpotter throwing issue

### ISSUE THREAD

**Issue by aishukc** [projectdiscovery/subfinder#200]
Running the tool as following throws error on certain domain. 
`subfinder -d domain.com`

```
panic: runtime error: index out of range [-1]

goroutine 36 [running]:
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter.(*Source).Run.func1(0xc0000d84d0, 0xc00017c180, 0x7ffd9b838027, 0xb, 0xd35280)
	/home/vps/go/src/github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter/certspotter.go:52 +0xc6e
created by github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter.(*Source).Run
	/home/vps/go/src/github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter/certspotter.go:23 +0x89


```

I think the error is thrown when certspotter does not return any data. 

## How can we reproduce the issue?
1. To reproduce the issue try running the tool against non existing domain  (issue also occurs with existing domain though)
`subfinder -d a33weasdlsdf23wedsaaaaaaaaa.com`

## What are the running context details?
* Installation method (e.g. `pip`, `apt-get`, `git clone` or `zip`/`tar.gz`):  go get -u -v github.com/projectdiscovery/subfinder/cmd/subfinder
* Client OS  - Linux
* Program version (see banner):  v2
* Relevant console output (if any):

**Comment by Ice3man543** [comment]
Hey @aishukc,
Nice catch. I'll see if i can reproduce and fix this asap.

**Comment by Ice3man543** [comment]
Yep, i was able to reproduce the issue.

```
panic: runtime error: index out of range

goroutine 66 [running]:
github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter.(*Source).Run.func1(0xc000138000, 0xc00019e000, 0xc000026fd0, 0xf, 0xd612a8)
        /root/go/src/github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter/certspotter.go:52 +0xce0
created by github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter.(*Source).Run
        /root/go/src/github.com/projectdiscovery/subfinder/pkg/subscraping/sources/certspotter/certspotter.go:23 +0x89
```

**Comment by Ice3man543** [comment]
Fixed in v2.2.3. Check it out now!
